server/caselist/caselistPresenter.ts

```diff
@@ -123,18 +124,45 @@ export default class CaselistPresenter {
     }
   }
 
-  searchByStatusArgs(): SelectArgs {
+  searchByStatusArgs(): {
```

Comment:
No need to change now. I've noticed the for the case list page we seem to have done a lot of the component setup in the presenter, rather than the view where it's usually done. Something we can maybe change at some point for consistency